traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -992,6 +992,9 @@ func Update(w http.ResponseWriter, r *http.Request) {
 			api.HandleErr(w, r, tx, http.StatusBadRequest, err, nil)
 			return
 		}
+		if newServer.XMPPID != nil {
+			fmt.Println("Change in xmpp_id was requested but it will be ignored to ensure hashId remains consistent")
```

Comment:
Change to `log.Debugln` or something like that

Reply:
Changed.